src/main/java/uk/gov/pay/ledger/transactionsummary/service/TransactionSummaryService.java

```diff
@@ -16,65 +16,69 @@
 import java.util.stream.Collectors;
 
 import static java.time.ZoneOffset.UTC;
-import static java.time.ZonedDateTime.parse;
 import static uk.gov.pay.ledger.event.model.SalientEventType.CAPTURE_CONFIRMED;
+import static uk.gov.pay.ledger.event.model.SalientEventType.CAPTURE_SUBMITTED;
 import static uk.gov.pay.ledger.event.model.SalientEventType.PAYMENT_CREATED;
 import static uk.gov.pay.ledger.event.model.SalientEventType.from;
-import static uk.gov.pay.ledger.transaction.model.TransactionType.PAYMENT;
 
 public class TransactionSummaryService {
 
     private final TransactionSummaryDao transactionSummaryDao;
-    private final EventService eventService;
+
+    private final List<String> ignoreEventsForTransactionAmount = List.of(CAPTURE_CONFIRMED.name(),
+            CAPTURE_SUBMITTED.name());
 
     @Inject
-    public TransactionSummaryService(TransactionSummaryDao transactionSummaryDao, EventService eventService) {
+    public TransactionSummaryService(TransactionSummaryDao transactionSummaryDao) {
         this.transactionSummaryDao = transactionSummaryDao;
-        this.eventService = eventService;
     }
 
-    public void projectTransactionSummary(TransactionEntity transaction, Event currentEvent) {
-        if (PAYMENT.name().equals(transaction.getTransactionType())) {
-            projectPaymentTransactionSummary(transaction, currentEvent);
+    public void projectTransactionSummary(TransactionEntity transaction, Event currentEvent, List<Event> events) {
+        if (TransactionType.PAYMENT.name().equals(transaction.getTransactionType())) {
+            projectPaymentTransactionSummary(transaction, currentEvent, events);
         }
     }
 
-    private void projectPaymentTransactionSummary(TransactionEntity transaction, Event currentEvent) {
-
+    private void projectPaymentTransactionSummary(TransactionEntity transaction, Event currentEvent,
+                                                  List<Event> events) {
         if (!canProjectTransactionSummary(transaction, currentEvent)) {
             return;
         }
 
-        List<Event> events = eventService.getEventsForResource(transaction.getExternalId());
-        boolean hasPaymentCreatedEvent = hasSalientEvent(events, PAYMENT_CREATED);
-
-        if (!hasPaymentCreatedEvent) {
+        if (!hasPaymentCreatedOrNotificationEvent(events)) {
             return;
         }
 
-        List<Event> eventsMappingToFinishedState =
-                getEventsMappingToTransactionFinishedStateInDescendingOrder(events);
-        Optional<SalientEventType> mayBeCurrentSalientEventType = from(currentEvent.getEventType());
+        if (!ignoreEventsForTransactionAmount.contains(currentEvent.getEventType())) {
```

Comment:
Is it worth adding a comment about why we ignore these payment types? From the code it isn't obvious so could trip someone up later.

Reply:
👍🏽, will add in next PR